plugins/nexus-repository-manager/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@janus-idp/backstage-plugin-nexus-repository-manager",
+  "version": "1.0.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.esm.js",
+    "types": "dist/index.d.ts"
+  },
+  "backstage": {
+    "role": "frontend-plugin"
+  },
+  "scripts": {
+    "start": "backstage-cli package start",
+    "pretsc": "openapi --input ./nexus-swagger.json --output ./src/generated --useUnionTypes --useOptions",
+    "build": "backstage-cli package build",
+    "lint": "backstage-cli package lint",
+    "test": "backstage-cli package test --passWithNoTests --coverage",
+    "clean": "backstage-cli package clean",
+    "prepack": "backstage-cli package prepack",
+    "postpack": "backstage-cli package postpack"
+  },
+  "dependencies": {
+    "@backstage/catalog-model": "^1.4.1",
+    "@backstage/core-components": "^0.13.3",
+    "@backstage/core-plugin-api": "^1.5.3",
+    "@backstage/plugin-catalog-react": "^1.8.0",
+    "@janus-idp/shared-react": "*",
```

Comment:
will this be `*`?

Reply:
Once [this](https://github.com/janus-idp/backstage-plugins/pull/633) is merged we can update the value here.